aspnet/hosting/servicing.rst

```diff
@@ -0,0 +1,34 @@
+.. _hosting-servicing:
+
+Servicing
+=========
+
+By `Sourabh Shirhatti`
+
+ASP.NET 5 supports servicing of runtime components (DNX) and packages through Windows Update Services which will deliver updates to patch any vulnerabilities if and when they are discovered. This document provides an overview of how to setup your Windows Server correctly to receive updates.
+
+Breadcrumbs Directory
+---------------------
+
+All serviceable assemblies will leave a breadcrumb in the ``BreadcrumbStore`` Directory. At the time of servicing Windows Update looks in this directory to figure out which assemblies are used on the server and need patching. This directory must be correctly protected by ACLs to prevent rogue applications from deleting entries from this directory. Run the powershell snippet below in an elevated prompt to create the ``BreadcrumbStore`` directory and ACL it correctly.
+
+
```

Comment:
Nits:
" At the time of servicing, Windows" (add comma)
"and need patching" perhaps to "and require patching"
"must be correctly protected" ... one wouldn't want to "incorrectly" do it, so recommend just "must be protected"
"ACLs" to "ACL's" ... but that's a style thing, and I don't know what spec you're using
"powershell" to "PowerShell"
"ACL" probably shouldn't be treated like a verb here. Maybe ... "and set ACL's on it correctly."